common/djangoapps/django_sudo_helpers/settings.py

```diff
@@ -0,0 +1,8 @@
+"""
+Settings for the django-sudo module.
+"""
+
+
+def apply_settings(django_settings):
+    """Set provider-independent settings."""
+    django_settings.SOCIAL_AUTH_PIPELINE += ('django_sudo_helpers.pipeline.create_sudo_session',)
```

Comment:
haha, sorry for the conflicting feedback then. A comment in the other file works for me!